### PR TITLE
Support nocopy option in volume binds (#688)

### PR DESCRIPTION
**Pull request by dmandalidis**
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/696)

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/model/Bind.java [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
     private AccessMode accessMode;
 
+    private boolean noCopy;
```
Comment (marcuslinke) [inline src/main/java/com/github/dockerjava/api/model/Bind.java]: please use Boolean here

**Comment by KostyaSha** [comment]
Please annotate field with `since` docker API version it appeared.

**Comment by dmandalidis** [comment]
...I think that the pr build is failing independently of this pr

**Comment by KostyaSha** [comment]
@dmandalidis is it possible to make some integration test?

**Comment by dmandalidis** [comment]
@KostyaSha done

File: src/test/java/com/github/dockerjava/netty/exec/CreateContainerCmdExecTest.java [hunk 2]
```diff
@@ -139,8 +143,42 @@ public void createContainerWithReadOnlyVolume() throws DockerException {
...
+        
+        assertThat(inspectContainerResponse1.getMounts().get(0).getDestination(), equalTo(volume1));
+        assertThat(inspectContainerResponse1.getMounts().get(0).getMode(), equalTo("rw,nocopy"));
+        assertThat(inspectContainerResponse1.getMounts().get(0).getRW(), equalTo(true));
+    }
+    
+    @Test
     public void createContainerWithVolumesFrom() throws DockerException {
-
+
```
Comment (KostyaSha) [inline src/test/java/com/github/dockerjava/netty/exec/CreateContainerCmdExecTest.java]: Please remove this whitespaces.

**Comment by KostyaSha** [comment]
Oh, this vs #554

**Pull request closed without merging**